dhis-2/dhis-api/src/test/java/org/hisp/dhis/period/RelativePeriodTest.java

```diff
@@ -294,7 +310,8 @@ public void testGetLast4Quarters()
     {
         RelativePeriods relativePeriods = new RelativePeriods().setLast4Quarters( true );
         
-        List<Period> relatives = relativePeriods.getRelativePeriods( getDate( 2001, 1, 1 ), I18N_FORMAT, false );
+        List<Period> relatives = relativePeriods.getRelativePeriods( getDate( 2001, 1, 1 ), I18N_FORMAT, false,
+            "FINANCIAL_PERIOD_OCTOBER" );
```

Comment:
Avoid having strings here, use enum or constants. And is it financial YEAR and not PERIOD?